### PR TITLE
Add Hibtc2 exchange in README and configuration.md

### PR DESCRIPTION
**Pull request by glonlas**
## Summary
Since HitBTC v2 was successfully tested by @ZeroCool940711. This PR add the exchange in the README and configuration.md.

Solve the issue: #842

## Quick changelog

- Add HitBTC v2 in the doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,10 @@ strategy parameters with real exchange data.
 ### Exchange marketplaces supported
 - [X] [Bittrex](https://bittrex.com/)
 - [X] [Binance](https://www.binance.com/)
-- [ ] [113 others to tests](https://github.com/ccxt/ccxt/). _(We cannot guarantee they will work)_
+- [X] [HitBTC v2](https://hitbtc.com/)
+- [ ] [112 others to tests](https://github.com/ccxt/ccxt/). _(We cannot guarantee they will work)_
+You have tried an exchange not listed here and it is working here?
```
Comment (vertti) [inline README.md]: remove the second "here".

File: docs/configuration.md [hunk 2]
```diff
@@ -83,11 +83,12 @@ end up paying more then would probably have been necessary.
 Freqtrade is based on [CCXT library](https://github.com/ccxt/ccxt) that supports 115 cryptocurrency
 exchange markets and trading APIs. The complete up-to-date list can be found in the
 [CCXT repo homepage](https://github.com/ccxt/ccxt/tree/master/python). However, the bot was tested
-with only Bittrex and Binance.
+with the exchange listed below.
```
Comment (vertti) [inline docs/configuration.md]: exchange -> exchanges

**Comment by xmatthias** [comment]
based on [this](https://github.com/freqtrade/freqtrade/issues/845#issuecomment-394710934) comment in #845 i would ask to wait for confirmation before merging this - as hitbtc seems to have ticker-issues ...

**Comment by ZeroCool940711** [comment]
I agree with @xmatthias , wait a bit more until those issues can be fixed, im trying to get as much information as I can to see if that can be fixed soon.

**Comment by xmatthias** [comment]
After anlyzing this further [here](https://github.com/freqtrade/freqtrade/issues/845#issuecomment-394828904) i think we run into [this](https://github.com/ccxt/ccxt/issues/2937) upstream bug affecting OHLCV data for some exchanges.

I don't think we can add hitBTC2 to the supported exchanges until this is fixed.

**Comment by glonlas** [comment]
Since this feature is not ready I will close the PR.

**Pull request closed without merging**